components/LoginForm/LoginForm.js

```diff
@@ -64,16 +69,18 @@ function LoginForm({ initialValues, login, onSuccess }) {
             <Field
               type="email"
               name="email"
-              label="Email*"
+              label={redirectFunc ? '' : 'Email*'}
               component={Input}
+              placeholder={redirectFunc ? 'Enter Your Username' : ''}
```

Comment:
While the wireframe had this, we can do without it and the other placeholder too.

Reply:
✔️